DuckDuckGo/TabViewControllerBrowsingMenuExtension.swift

```diff
@@ -298,13 +298,15 @@ extension TabViewController {
             Pixel.fire(pixel: .emailUserCreatedAlias, withAdditionalParameters: pixelParameters, includedParameters: [])
 
             emailManager.getAliasIfNeededAndConsume { alias, _ in
-                guard let alias = alias else {
-                    // we may want to communicate this failure to the user in the future
-                    return
+                Task { @MainActor in
```

Comment:
Since all the code contained within this closure is meant to update the UI in one way or another, let's force execution in `MainActor`.